### PR TITLE
palava-web role: Install nginx & node

### PR DESCRIPTION
**Pull request by janlelis**

### CODE REVIEW

File: playbooks/web.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (erikzenker) [inline playbooks/web.yml]: Should be call the file something like `install_palava_web.yml` to be consistent with `install_signaltower`

File: roles/requirements.yml [hunk 2]
```diff
@@ -2,4 +2,4 @@
 - name: cloudalchemy.prometheus
 - name: cloudalchemy.grafana
 - name: geerlingguy.certbot
-- name: nginxinc.nginx
\ No newline at end of file
+- name: community.general
```
Comment (erikzenker) [inline roles/requirements.yml:5]: What do we need that for?

**Review by erikzenker — APPROVED** [review]
Thanks for your contribution. Did you smoke tested it on our new server?

File: playbooks/web.yml (hunk 1 above)
Comment (janlelis) [reply playbooks/web.yml]: right, good point, although let's maybe remove the install_ prefix, so it is:

- signaltower.yml
- palava_web.yml

What do you think about this? @erikzenker @juwi

File: roles/requirements.yml (hunk 2 above)
Comment (janlelis) [reply roles/requirements.yml:5]: For the [npm module](https://github.com/juwi/palava-ansible/pull/12/files#diff-7bcd639a9a366b7a7709700e015e9e9fac9a474e7418520d77507ef10fdab4b3R17)

**Comment by erikzenker** [comment]
Would be nice if you could Update the readme about how to install palava web

File: playbooks/web.yml (hunk 1 above)
Comment (erikzenker) [reply playbooks/web.yml]: I dont really care. Important for me is the consistency and that the usage is documented.

Comment (erikzenker) [reply playbooks/web.yml]: The `install` prefix suggest that the playbook can only install. If we drop the prefix then it should also be capable of updating etc. (general role)

File: roles/requirements.yml (hunk 2 above)
Comment (juwi) [inline roles/requirements.yml:5]: Sollten wir auch in die Readme aufnehmen dass das mit der Galaxy angezogen wird und vielleicht ein Satz dazu was das tut.

File: roles/palava-web/files/nodesource.gpg.key [hunk 3]
```diff
@@ -0,0 +1,52 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```
Comment (juwi) [inline roles/palava-web/files/nodesource.gpg.key:1]: Does this make sense? Much rather apt_key should be used to retrieve and install this key.

File: playbooks/web.yml (hunk 1 above)
Comment (erikzenker) [reply playbooks/web.yml]: I will just add the `install` prefix

File: roles/palava-web/files/nodesource.gpg.key (hunk 3 above)
Comment (juwi) [reply roles/palava-web/files/nodesource.gpg.key:1]: Disregard get_url. apt_key would be the right mechanism to use!

Comment (erikzenker) [reply roles/palava-web/files/nodesource.gpg.key:1]: I used the url field in the apt_key command of ansible and set it to `https://deb.nodesource.com/gpgkey/nodesource.gpg.key`

File: roles/requirements.yml (hunk 2 above)
Comment (erikzenker) [reply roles/requirements.yml:5]: I added it to the readme and put `(node)` as description for what we need it.

**Pull request merged**